Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -0,0 +1,455 @@
+
+        initialize(this, arrayBuffer, byteOffset);
+    }
+
+    // This can be overridden for testing purposes
+    Batched3DModel3DTileContent._deprecationWarning = deprecationWarning;
+
+    defineProperties(Batched3DModel3DTileContent.prototype, {
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        featuresLength : {
+            get : function() {
+                return this._batchTable.featuresLength;
+            }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        pointsLength : {
+            get : function() {
+                return 0;
+            }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        trianglesLength : {
+            get : function() {
+                return this._model.trianglesLength;
+            }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        geometryByteLength : {
+            get : function() {
+                return this._model.geometryByteLength;
+            }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        texturesByteLength : {
+            get : function() {
+                return this._model.texturesByteLength;
+            }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        batchTableByteLength : {
+            get : function() {
+                return this._batchTable.memorySizeInBytes;
+            }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        innerContents : {
+            get : function() {
+                return undefined;
+            }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        readyPromise : {
+            get : function() {
+                return this._model.readyPromise;
+            }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        tileset : {
+            get : function() {
+                return this._tileset;
+            }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        tile : {
+            get : function() {
+                return this._tile;
+            }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        url: {
+            get: function() {
+                return this._url;
+            }
+        },
+
+        /**
+         * Part of the {@link Cesium3DTileContent} interface.
+         */
+        batchTable : {
+            get : function() {
+                return this._batchTable;
+            }
+        }
+    });
+
+    var sizeOfUint32 = Uint32Array.BYTES_PER_ELEMENT;
```

Comment:
Since this is trying to use `Uint32Array` in a global scope, you need to add a check for typed arrays at the top of this file, otherwise it will fail to even load `Cesium.js` in unsupported browsers (which means users can't fallback to other engines).  We usually use

```
    if (!FeatureDetection.supportsTypedArrays()) {
        return {};
    }
```

right after `use strict`, see `ComponentDataType.js` for an example.

To test this, set IE11 to "IE9" mode in the emulation tab of the debugger, and then load `http://localhost:8080/Specs/SpecRunner.html?category=none`.  You'll see errors everywhere we are using a typed array.  Each file that triggers it should have the above check.